core/src/test/scala/org/apache/spark/deploy/history/EventLogTestHelper.scala

```diff
@@ -38,6 +38,7 @@ object EventLogTestHelper {
   def getLoggingConf(logDir: Path, compressionCodec: Option[String] = None): SparkConf = {
     val conf = new SparkConf
     conf.set(EVENT_LOG_ENABLED, true)
+    conf.set(EVENT_LOG_ENABLE_ROLLING, false)
```

Comment:
This is consistent with the existing function description.

https://github.com/apache/spark/blob/5970d353360d4fb6647c8fbc10f733abf009eca1/core/src/test/scala/org/apache/spark/deploy/history/EventLogTestHelper.scala#L35